bot/admin/admin.py

```diff
@@ -453,18 +454,16 @@ async def botonly(self, ctx: commands.Context, channel: Optional[discord.TextCha
         await target_channel.send(embed=embed)
 
     @commands.Cog.listener(name='on_message')
-    async def on_message(self, ctx: discord.Message):
+    async def on_message(self, message: discord.Message):
```

Comment:
I support it because we often use the name `ctx` when we have an object of the data type `discord.ext.commands.Context`. This listener explicitly gets a object of the class `discord.Message` so it would be the best not to confuse these two and simply name it message, because that is exactly what you get.

Reply:
I always used it there for the idea of the command context, no matter if it was a Context object.